packages/gestalt/src/Button.js

```diff
@@ -44,7 +44,7 @@ type BaseButton = {|
   >,
   size?: 'sm' | 'md' | 'lg',
   text: string,
-  textColor?: 'white' | 'darkGray' | 'blue' | 'red',
+  textColor?: 'white',
```

Comment:
@AlbertCarreras why do we have to keep this?

Reply:
![image](https://user-images.githubusercontent.com/10593890/92269823-cf16b400-ee99-11ea-8653-f39cc23d609f.png)
![image](https://user-images.githubusercontent.com/10593890/92269846-d3db6800-ee99-11ea-813c-3be8217ad2df.png)

Reply:
I'm gonna document